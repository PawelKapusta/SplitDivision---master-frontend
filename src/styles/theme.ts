import { DefaultTheme } from "styled-components";

export const theme: DefaultTheme = {
    palette: {
        white: "#fff",
        black: "#000",
        background1: "#0F1624",
        gold: "#eaaf57",
    },
    breakpoints: {
        sm: "screen and (max-width: 640px)",
        md: "screen and (max-width: 768px)",
        lg: "screen and (max-width: 1024px)",
        xl: "screen and (max-width: 1280px)",
    },
};
