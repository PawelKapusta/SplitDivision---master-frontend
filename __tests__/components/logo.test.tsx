import { render } from "@testing-library/react";
import Logo from "@components/logo/index";
import { mockTheme } from "../mocks/theme";
import { ThemeProvider } from "styled-components";

describe("logo component", () => {
    it("renders correctly", () => {
        const { container } = render(
            <ThemeProvider theme={mockTheme}>
                <Logo />
            </ThemeProvider>,
        );
        expect(container).toMatchSnapshot();
    });
});
