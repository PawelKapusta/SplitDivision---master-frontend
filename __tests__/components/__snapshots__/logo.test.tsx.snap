// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logo component renders correctly 1`] = `
<div>
  <div
    class="sc-61620a93-0 kYZJFy"
  >
    <img
      alt="logoImage"
      data-nimg="1"
      decoding="async"
      height="95"
      loading="lazy"
      src="/_next/image?url=%2Fimages%2Flogo.png&w=384&q=75"
      srcset="/_next/image?url=%2Fimages%2Flogo.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2Flogo.png&w=384&q=75 2x"
      style="color: transparent;"
      width="175"
    />
  </div>
</div>
`;
